pkg/algorithm/hived_algorithm.go

```diff
@@ -151,44 +157,56 @@ func (h *HivedAlgorithm) Schedule(pod *core.Pod, suggestedNodes []string) intern
 
 	klog.Infof("[%v]: Scheduling pod...", internal.Key(pod))
 	s := internal.ExtractPodSchedulingSpec(pod)
-	// gpu number -> a set of pods -> a set of GPUs of each pod
-	groupPhysicalPlacement := groupPhysicalPlacement{}
-	groupVirtualPlacement := groupVirtualPlacement{}
-	podIndex := int32(0)
 	suggestedNodeSet := common.NewSet()
 	for _, n := range suggestedNodes {
 		suggestedNodeSet.Add(n)
 	}
-
-	group := h.allocatedAffinityGroups[s.AffinityGroup.Name]
-	if group == nil {
+	var (
+		groupPhysicalPlacement groupPhysicalPlacement // GPU number -> a set of pods -> a set of GPUs of each pod
+		groupVirtualPlacement  groupVirtualPlacement  // GPU number -> a set of pods -> a set of GPUs of each pod
```

Comment:
Move these comment to:
type groupPhysicalPlacement map[int32][]CellList
type groupVirtualPlacement map[int32][]CellList

Reply:
So you can remove it here?

---
In reply to: [397593293](https://github.com/microsoft/hivedscheduler/pull/8#discussion_r397593293) [](ancestors = 397593293)